leanstacks/react-common
# Storybook

## :pick: Change Summary

- add storybook
- integrate storybook and tailwind
- add support for light and dark tailwind themes
- add stories for `Text` component

## :test_tube: Steps to Test

1. test running storybook locally
2. ensure autodocs generated successfully
3. review all stories
4. test running storybook build; ensure build is successful
5. ensure component build is successful
6. publish and test new version of component

## :link: Additional Information

- ...
